src/core/client/stream/mutations/CreateCommentMutation.ts

```diff
@@ -102,9 +153,13 @@ function commit(
       },
     } as any, // TODO: (cvle) generated types should contain one for the optimistic response.
     optimisticUpdater: store => {
+      sharedUpdater(store, input);
       store.get(id)!.setValue(true, "pending");
     },
-    configs: getConfig(input),
+    updater: store => {
+      sharedUpdater(store, input);
+    },
+    // configs: getConfig(input),
```

Comment:
Can you remove this line if if isn't in use?

Reply:
Oh sure, forgot that one